### PR TITLE
Maybe the real algorithm was inside us all along

### PR DESCRIPTION
**Pull request by AlenVelocity**

### CODE REVIEW

**Comment by kenluzuriaga0** [comment]
so deep!  bro

File: README.md [hunk 1]
```diff
@@ -1,39 +1,3 @@
...
-|                    | [twml](twml/README.md) | Legacy machine learning framework built on TensorFlow v1. |
-
-We include Bazel BUILD files for most components, but not a top level BUILD or WORKSPACE file.
-
-## Contributing
-
-We invite the community to submit GitHub issues and pull requests for suggestions on improving the recommendation algorithm. We are working on tools to manage these suggestions and sync changes to our internal repository. Any security concerns or issues should be routed to our official [bug bounty program](https://hackerone.com/twitter) through HackerOne. We hope to benefit from the collective intelligence and expertise of the global community in helping us identify issues and suggest improvements, ultimately leading to a better Twitter.
-
-Read our blog on the open source initiative [here](https://blog.twitter.com/en_us/topics/company/2023/a-new-era-of-transparency-for-twitter).
+Maybe the real algorithm was the friends we made along the way
```
Comment (shubhamdhingra38) [inline README.md:3]: 🥺
LGTM

**Comment by AliAryanTech** [comment]
Indeed, they should accept this pull request for sure!!

**Comment by 024x** [comment]
> Indeed, they should accept this pull request for sure!! 

Yes!

**Pull request closed without merging**